docs/components/DropdownButtonViewData.jsx

```diff
@@ -0,0 +1,254 @@
+  RS: {country: "Serbia", hello: "Zdravo"},
+  RU: {
+    country: "Russian Federation",
+    hello: (
+      <span>
+        &#1047;&#1076;&#1088;&#1072;&#1074;&#1089;&#1090;&#1074;&#1091;&#1081;&#1090;&#1077;
+      </span>
+    ),
+  },
+  RW: {country: "Rwanda", hello: "Hello"},
+  SA: {country: "Saudi Arabia", hello: "Marhaba"},
+  SB: {country: "Solomon Islands", hello: "Hello"},
+  SC: {country: "Seychelles", hello: "Hello"},
+  SD: {country: "Sudan", hello: "Marhaba"},
+  SE: {country: "Sweden", hello: "God dag"},
+  SG: {country: "Singapore", hello: "Selamat"},
+  SI: {country: "Slovenia", hello: "Živijo"},
+  SK: {country: "Slovakia", hello: "Dobrý deň"},
+  SL: {country: "Sierra Leone", hello: "Hello"},
+  SM: {country: "San Marino", hello: "Buon giorno"},
+  SN: {country: "Senegal", hello: "Bonjour"},
+  SO: {country: "Somalia", hello: "Maalim wanaqsan"},
+  SR: {country: "Suriname", hello: "Hallo"},
+  ST: {country: "Sao Tome and Principe", hello: "Hello"},
+  SV: {country: "El Salvador", hello: "Hola"},
+  SY: {country: "Syrian Arab Republic", hello: "Marhaba"},
+  SZ: {country: "Swaziland", hello: "Hello"},
+  TC: {country: "Turks and Caicos Islands", hello: "Hello"},
+  TD: {country: "Chad", hello: "Marhaba"},
+  TG: {country: "Togo", hello: "Bonjour"},
+  TH: {country: "Thailand", hello: "Sawatdi"},
+  TJ: {country: "Tajikistan", hello: "Salom"},
+  TK: {country: "Tokelau", hello: "Taloha"},
+  TM: {country: "Turkmenistan", hello: "Salam"},
+  TN: {country: "Tunisia", hello: "Marhaba"},
+  TO: {country: "Tonga", hello: "Malo e lelei"},
+  TR: {country: "Turkey", hello: "Merhaba"},
+  TT: {country: "Trinidad and Tobago", hello: "Hello"},
+  TV: {country: "Tuvalu", hello: "Talofa"},
+  TW: {country: "Taiwan", hello: <span>&#20320;&#22909;</span>},
+  TZ: {country: "Tanzania  United Republic of", hello: "Habari"},
+  UA: {country: "Ukraine", hello: "Pryvit"},
+  UG: {country: "Uganda", hello: "Habari"},
+  UK: {country: "United Kingdom", hello: "Hello"},
+  UM: {country: "United States Minor Outlying Islands", hello: "Hello"},
+  US: {country: "United States", hello: "Hello"},
+  UY: {country: "Uruguay", hello: "Hola"},
+  UZ: {country: "Uzbekistan", hello: "Salom"},
+  VA: {country: "Holy See (Vatican City State)", hello: "Buon giorno"},
+  VC: {country: "Saint Vincent and the Grenadines", hello: "Hello"},
+  VE: {country: "Venezuela", hello: "Hola"},
+  VG: {country: "Virgin Islands  British", hello: "Hello"},
+  VI: {country: "Virgin Islands  U.S.", hello: "Hello"},
+  VN: {country: "Vietnam", hello: "Chào"},
+  VU: {country: "Vanuatu", hello: "Halo"},
+  WF: {country: "Wallis and Futuna", hello: "Malo le kataki"},
+  WS: {country: "Samoa", hello: "Talofa"},
+  YE: {country: "Yemen", hello: "Marhaba"},
+  YT: {country: "Mayotte", hello: "Hello"},
+  ZA: {country: "South Africa", hello: "Hello"},
+  ZM: {country: "Zambia", hello: "Hello"},
+  ZW: {country: "Zimbabwe", hello: "Hello"},
+};
```

Comment:
hopefully, we don't get called out for mistakes. i just picked the first random google result for "hello in different languages json" 😁